content/publish.sh

```diff
@@ -14,9 +14,13 @@
 # limitations under the License.
 
 # Publishes content files to Google Cloud Storage as publicly accessible content.
-
 # Synchronize content from the repository content directory to the
 # cloud within gs://cloud-datalab/content
+
+# First delete all content because rsync may not always override destination objects.
+gsutil -m rm -r -f gs://cloud-datalab/content
```

Comment:
Did some experiments and it seems cp does not delete objects that we want to remove. Emptying the bucket seems cleaner?